recipes/lapack95/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "lapack95" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.netlib.org/lapack95/lapack95.tgz
+  sha256: 55711e5b1c5ddc9c7dd1d3ea15c37fc5b400783ac6ea573a75e2b1358c2c479b
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+  host:
+    - openblas
+    - libblas * *openblas
+    - liblapack * *openblas
+  run:
+    - libopenblas
```

Comment:
```suggestion
  host:
    - libblas
    - liblapack
```

Reply:
@isuruf if I remove openblas it will install the `netlib` alternative of libblas and therefore I will go back into the issue that I had from the missing symbols. 
More details here: https://github.com/conda-forge/staged-recipes/pull/14212#issuecomment-797701592

Reply:
which symbols are missing?

Reply:
`_clagge_` is one of them... I can apply your patch and get a full list.

Reply:
If you try this:
`nm $PREFIX/lib/liblapack.dylib | grep clagge` on a MacOS machine with lapack from netlib or blis it won't work.. but it does for mkl and openblas.

Reply:
I pushed a change so it can run the CI without openblas and all the missing symbols will show. My debug will also print the output of `nm` for liblapack.dylib to confirm the missing symbols.

Reply:
Ah, the testing symbols are missing. Let me fix that.

Reply:
Awesome! Thanks

Reply:
@isuruf here is the full list of missing symbols:
```
Undefined symbols for architecture x86_64:
  "_clagge_", referenced from:
      _clagge_f95_ in lapack95.a(la_clagge.o)
     (maybe you meant: _clagge_f95_)
  "_dlagge_", referenced from:
      _dlagge_f95_ in lapack95.a(la_dlagge.o)
     (maybe you meant: _dlagge_f95_)
  "_slagge_", referenced from:
      _slagge_f95_ in lapack95.a(la_slagge.o)
     (maybe you meant: _slagge_f95_)
  "_zlagge_", referenced from:
      _zlagge_f95_ in lapack95.a(la_zlagge.o)
     (maybe you meant: _zlagge_f95_)
ld: symbol(s) not found for architecture x86_64
```


More details, including `nm liblapack.dylib` here:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=290081&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=3595